ibtk/include/ibtk/LibMeshSystemVectors.h

```diff
@@ -158,14 +134,17 @@ class LibMeshSystemVectors
     void zero(const std::string& vec_name);
 
 protected:
-    /// Check if a vector exists and, if not, add it.
-    void maybeAdd(const std::string& vec_name);
+    LibMeshSystemVectors(const std::vector<bool>& part_mask, std::string system_name);
```

Comment:
As mentioned below, we should not have this constructor since it doesn't fully construct the object.